server/src/database/scripts/sync-chefs-data.service.ts

```diff
@@ -0,0 +1,80 @@
+import axios from 'axios';
+import { Injectable } from '@nestjs/common';
+import { Repository } from 'typeorm';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Application } from '../../application/application.entity';
+import { CHEFS_BASE_URL, CHEFS_FORM_IDS, REQUEST_METHODS } from '../../common/constants';
+import { ApplicationService } from '@/application/application.service';
+import { SaveApplicationDto } from '@/common/dto/save-application.dto';
+import { ReviewStatuses } from '@/common/enums';
+
+@Injectable()
+export class SyncChefsDataService {
+  constructor(
+    @InjectRepository(Application)
+    private readonly applicationRepo: Repository<Application>
+  ) {}
+
+  private getFormUrl(form) {
```

Comment:
type form to string and instead use `formId`

Reply:
Makes sense, thanks Karthikk!